src/fluree/db/api_js.cljs

```diff
@@ -632,10 +632,10 @@
                                             {:status 400 :error :db/invalid-signed-query}))))
          sig         (crypto/sign-message cmd private-key)
          id          (crypto/sha3-256 cmd)]
-     {:cmd cmd
-      :sig sig
-      :id  id
-      :db  ledger})))
+     {:cmd    cmd
+      :sig    sig
+      :id     id
+      :ledger ledger})))
 
```

Comment:
These command maps are used downstream by the transactors... I can't recall but just to confirm you've adjusted the transactors to look now for :ledger when processing commands too?

Probably not worth worrying about, but if there were any queued commands... or any other libraries sending the format with :db ... it will break the transactors. Might be a good idea to support either/or :db and :ledger for the interim.

I'm OK with letting it go... but worth keeping ears open in the short-term in case anyone has an issue.

Reply:
Yeah, pretty much everywhere else has been changed to generate and expect `:ledger` in these (unless there are others I've somehow missed, but hopefully these were the last of them). The v1 code should still allow either `:db` or `:ledger`.